doc/Standaard-inrichting-GitHub-GitLab.md

```diff
@@ -170,6 +186,7 @@ Zodra het product waarvoor de repository in het leven is geroepen een bepaalde s
 Meer informatie over het activeren van GitHub Pages specifiek voor VNG Realisatie vind je in [de GitHub Pages sectie van de API Kennisbank](https://github.com/VNG-Realisatie/API-Kennisbank/blob/master/GitHub%20Pages/GitHub-Pages-implementatie-VNG-R.md).
 
 ## GitLab
+OPMERKING: Gitlab kende indertijd een beter CI/CD integratie en de mogelijkheid met private repositories te werken. Inmiddels kent Github ook private repos en wellicht is de CI/CD aansluiting ook verbeterd. De vraag is dus in hoeverre we nog meerdere git omgevingen willen ondersteunen? Het maakt het wel weer minder overzichtelijk en transparant.
```

Comment:
Helemaal mee eens. Ik weet niet of het aan ons is te bepalen of we doorgaan met GitLab. Ik wil wel voorstellen dat we hier heel kort iets zeggen over GitLab maar dat we dit tool binnen ons Kenniscentrum zeer beperkt gebruiken en dat we daarover hier niet verder uitweiden.